pkg/apis/componentconfig/types.go

```diff
@@ -230,7 +230,7 @@ type KubeletConfiguration struct {
 	// be rejected.
 	LowDiskSpaceThresholdMB int `json:"lowDiskSpaceThresholdMB"`
 	// How frequently to calculate and cache volume disk usage for all pods
-	VolumeStatsAggPeriod unversioned.Duration `json:volumeStatsAggPeriod`
+	VolumeStatsAggPeriod unversioned.Duration `json:"volumeStatsAggPeriod"`
```

Comment:
Change looks good to me. Lower case is consistent with the rest of the file so I would leave it. This is not used yet and IIUC json tags aren't necessary in the internal types.go so we may remove them later. Thanks!